www/js/diary/list.js

```diff
@@ -277,14 +273,44 @@ angular.module('emission.main.diary.list',['ui-leaflet',
             DiaryHelper.directiveForTrip);
           Timeline.setTripWrappers(currDayTripWrappers);
 
+          let tripFromNotification = null;
+
           $scope.data.currDayTripWrappers.forEach(function(tripgj, index, array) {
             $scope.populateModeFromTimeline(tripgj, $scope.data.unifiedConfirmsResults.modes);
             $scope.populatePurposeFromTimeline(tripgj, $scope.data.unifiedConfirmsResults.purposes);
             $scope.populateSurveyAnswerFromTimeline(tripgj, $scope.data.unifiedConfirmsResults.surveyAnswers);
             $scope.populateBasicClasses(tripgj);
             $scope.populateCommonInfo(tripgj);
           });
+          if($rootScope.displayingIncident == true && $rootScope.notificationData) {
+            const matchingTrips = $scope.data.currDayTripWrappers.filter(trip => {
```

Comment:
arrow functions are not supported in iOS9 - they result in a WSOD. Please switch to explicit `function(trip)` unless you plan to publish your own app to the store and set the min iOS version to a higher value. the default emission app supports older versions of both android and iOS for maximum compatibility.

You can test this using the emulator as well.
https://webkit.org/blog/4054/es6-in-webkit/